libraries/chain/database.cpp

```diff
@@ -3011,11 +3027,18 @@ namespace golos { namespace chain {
                 create<account_object>([&](account_object &a) {
                     a.name = STEEMIT_MINER_ACCOUNT;
                 });
-#ifndef IS_LOW_MEM
-                create<account_metadata_object>([&](account_metadata_object& m) {
-                    m.account = STEEMIT_MINER_ACCOUNT;
-                });
-#endif
+
+                if (store_metadata_for_account(STEEMIT_MINER_ACCOUNT)) {
+                    create<account_metadata_object>([&](account_metadata_object& m) {
+                        m.account = STEEMIT_MINER_ACCOUNT;
+                    });
+                } else {
+                    auto meta = find<account_metadata_object, by_account>(STEEMIT_MINER_ACCOUNT);
+                    if (meta != nullptr) {
+                        remove(*meta);
+                    }
+                }
```

Comment:
Genesis state is initialized on create of the shared memory.
It is impossible that it will be called more than one time.

Remove `else {...}`